ramls/patron.raml

```diff
@@ -61,6 +61,21 @@ traits:
             required: false
             type: boolean
             default: false
+          offset:
+            description: Skip over a number of elements by specifying an offset value for the query
+            type: integer
+            required: false
+            example: 0
+            default: 0
+            minimum: 0
+            maximum: 2147483647
+          limit:
+            description: Limit the number of elements returned in the response
+            type: integer
+            required: false
+            example: 10
+            default: 2147483647
+            maximum: 2147483647
```

Comment:
Is there a reason you don't specify a minimum like we do above for offset?  Would this cause a change in behavior, or just would it only cause parameter validation to fail if limit < 0?